17.11 Redis docs (#307)
Macrometacorp/docs: docs/collections/redis-mode/_add-redis-document.md

---
title: Add Documents to Collection
sidebar_position: 30
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

This page explains how to add a document to a Document Store collection.

> By default, a new document has only one attribute, `_key`. Defining the document `_key` attribute is optional. If you do not specify a key, then it will be autogenerated.

Reviewer: ```suggestion
By default, a new document has only one attribute, `_key`. Defining the document `_key` attribute is optional. If you do not specify a key, then one is autogenerated when you add the document.
```

Wanted to adjust this to make it a bit more specific, but not sure if it's correct or really even necessary.  @diana-macrometa @dlozina-macrometa
